README.md

```diff
@@ -12,26 +12,33 @@ Google Cloud's container platforms:
 ## Quickstart
 
 1. [Install Docker](https://store.docker.com/search?type=edition&offering=community)
-1. [Install the pack tool (a CLI for running Buildpacks)](https://buildpacks.io/docs/install-pack/)
-1. Clone the [sample apps](https://github.com/GoogleCloudPlatform/buildpack-samples):
+2. [Install the pack tool (a CLI for running Buildpacks)](https://buildpacks.io/docs/install-pack/)
```

Comment:
Can you keep using `1.` throughout? No need to manually increment.

Reply:
eI can submit a new PR to make everything 1.